addsvc/grpc_binding.go

```diff
@@ -20,15 +20,15 @@ type grpcBinding struct{ server.Endpoint }
 // way to manipulate the RPC context, like headers for HTTP. So we don't have
 // a way to transport e.g. Zipkin IDs with the request. TODO.
 func (b grpcBinding) Add(ctx context.Context, req *pb.AddRequest) (*pb.AddReply, error) {
```

Comment:
Yes, we'll definitely add the traceID/spanID/parentSpanIDs to the context. But we'll probably need to make the keys configurable, as only Zipkin-over-HTTP has well-defined keys ([these](https://sourcegraph.com/github.com/go-kit/kit@8662a12630268cc431ff3825eb46f49a0719a894/.tree/tracing/zipkin/zipkin.go#startline=23&endline=26)), to the best of my knowledge.

> 'm having a little trouble getting the forked version of the repo running as all the include paths point the mainline.

The way to fork in Go is to add your fork as a git remote to the canonical checkout. That is,

```
$ cd $GOPATH/src/github.com/go-kit/kit
$ git remote add harlow git@github.com:harlow/kit
$ git checkout -b my-feature-branch
$ # commit commit commit
$ git push harlow my-feature-branch
```